### PR TITLE
Similarity Function Examples

### PR DESCRIPTION
**Pull request by mneedham**

### CODE REVIEW

**Comment by mneedham** [comment]
@davidoliverSP2 figured I may as well add similar examples for the other similarity functions at the same time as for Pearson

File: doc/asciidoc/similarity-pearson.adoc [hunk 1]
```diff
@@ -66,6 +66,47 @@ include::scripts/similarity-pearson.cypher[tag=function]
...
+[opts="header"]
+|===
+| `from`       | `to`         | `similarity`
+|"Arya" |	"Karin"	 | 0.8194651785206903
+
+|===
+// end::function-cypher[]
+
+
+In this example we pass in `vectorType: "maps"` as an extra parameter, as well as using the `algo.similarity.asVector` function to construct a vector of maps containing each movie and the corresponding rating.
```
Comment (davidoliverSP2) [inline doc/asciidoc/similarity-pearson.adoc]: missing comma.....

> In this example, .....

File: doc/asciidoc/similarity-pearson.adoc [hunk 2]
```diff
@@ -66,6 +66,47 @@ include::scripts/similarity-pearson.cypher[tag=function]
...
+We do this because the Pearson similarity algorithm needs to compute the average of *all* the movies that a user has reviewed, not just the ones that they have in common with the user we're comparing them to.
```
Comment (davidoliverSP2) [inline doc/asciidoc/similarity-pearson.adoc]: Algo name should be capitalised as per the header: **Pearson Similarity algorithm**

**Comment by mneedham** [comment]
@davidoliverSP2 cool! T'is merged in now

**Pull request merged**